QSB/TimeSync/Patches/TimePatches.cs

```diff
@@ -11,20 +11,56 @@ namespace QSB.TimeSync.Patches;
 [HarmonyPatch]
 public class TimePatches : QSBPatch
 {
-	public override QSBPatchTypes Type => QSBPatchTypes.OnClientConnect;
+	public override QSBPatchTypes Type => QSBPatchTypes.OnModStart;
 
-	/// <summary>
-	/// prevents wakeup prompt since we automatically wake you up.
-	/// (doesn't happen for host because we don't patch until TimeLoop._initialized i.e. after Start)
```

Comment:
so if this doesnt happen for the host then uh wont this not work

Reply:
nope, i fixed that by making it OnModStart